python-embedder/embedder.py

```
@@ -0,0 +1,45 @@
+from flask import Flask, request, jsonify
+from sentence_transformers import SentenceTransformer
+
+app = Flask(__name__)
+
+try:
+    model = SentenceTransformer('all-mpnet-base-v2')
+    MODEL_DIMENSIONS = model.get_sentence_embedding_dimension()
+except Exception as e:
```

Ruff (0.14.4)

Do not catch blind exception: `Exception`

(BLE001)